docs/index.html

```diff
@@ -40,11 +41,11 @@ <h1 class="text-center">EasySpeech</h1>
 </div>
 
 <div class="containter text-center">
-    <a href="https://github.com/jankapunkt/easy-speech/blob/master/API.md">API Doc</a>
-    <span>|</span>
-    <a href="https://github.com/jankapunkt/easy-speech">GitHub</a>
+    <button type="button" class="btn btn-primary"><a href="https://github.com/jankapunkt/easy-speech/blob/master/API.md"  class="text-decoration-none text-black ">API Doc</a></button>
```

Comment:
In bootstrap you should style links with the btn classes instead of wrapping them inside buttons:

```html
<a class="btn btn-primary"></a>
```

see: https://getbootstrap.com/docs/5.2/components/buttons/#button-tags